### PR TITLE
Add new sponsor, add twitter handles, and add speaker placeholder

### PR DESCRIPTION
**Pull request by missingdink**
:tennis:

### CODE REVIEW

File: source/index.haml [hunk 1]
```diff
@@ -81,25 +85,50 @@
...
+
         %li.rob-harr
-          %a.hover
+          .hover
             .overlay
               .text
                 Sparkbox
           %h2 Rob Harr
+          .twitter-handle
+            %a(href="http://www.twitter.com/robertharr/")
```
Comment (itsthatguy) [inline source/index.haml]: This could be on one line
`%a(href="http://www.twitter.com/robertharr/") @robertharr`

File: source/index.haml [hunk 2]
```diff
@@ -81,25 +85,50 @@
...
                 Northwest Independent Ruby Developers
           %h2 Renée Hendricksen
+          .twitter-handle
+            %a(href="http://www.twitter.com/nirdllc/")
+              @NIRDLLC
+
+        %li.locked
+          .hover
+            .overlay
+              .text
```
Comment (itsthatguy) [inline source/index.haml]: do we need the `.text` div if we don't have text?

File: source/stylesheets/shared/_base.scss [hunk 3]
```diff
@@ -136,7 +136,7 @@ $small-up: "only screen and (min-width: #{$small-screen}) and (max-width: 986px)
 @mixin portrait($image-url) {
   position: relative;
   &.boss {
-    > a:not(.button) {
+    > .hover {
       &:hover { @extend .shake-side; }
       @extend %boss-portrait;
       background-image: image-url("people/frank-byrne.jpg") !important;
```
Comment (itsthatguy) [inline source/stylesheets/shared/_base.scss]: let's use the locked image here, just to remove reference to frank-byrne

**Comment by missingdink** [comment]
Donezies :hankey:

**Comment by itsthatguy** [comment]
:+1: lgtm

**Pull request merged**